neo/Consensus/ConsensusContext.cs

```diff
@@ -29,13 +29,47 @@ internal class ConsensusContext : IDisposable
         public UInt160 NextConsensus;
         public UInt256[] TransactionHashes;
         public Dictionary<UInt256, Transaction> Transactions;
-        public byte[][] Signatures;
+        public byte[][] SignedPayloads;
+        public byte[][] FinalSignatures;
         public byte[] ExpectedView;
         private KeyPair KeyPair;
         private readonly Wallet wallet;
 
         public int M => Validators.Length - (Validators.Length - 1) / 3;
 
+        /// <summary>
+        /// Local PrepareRequest original Payload
+        /// </summary>
+        public ConsensusPayload PreparePayload;
+
+        /// <summary>
+        /// Serialize PreparePayload Data into the desired PrepareRequest message
+        /// </summary>
+        public PrepareRequest GetPrepareRequestMessage(ConsensusPayload PreparePayloadToGet)
```

Comment:
The first char of arguments in lowercase please :P

Reply:
I see, thanks, Shargon. Let's follow this standard and modify all cases.